qpc/insights/login_config.py

```diff
@@ -0,0 +1,45 @@
+"""AddLoginCommand is used to save insights login configuration."""
+
+from __future__ import print_function
+
+import qpc.insights as insights
+from qpc import messages
+from qpc.clicommand import CliCommand
+from qpc.translation import _
+from qpc.utils import write_insights_login_config
+
+
+# pylint: disable=too-few-public-methods
+class AddLoginCommand(CliCommand):
+    """Adds insights login information.
+
+    This command is for storing insights
+    login info, username and password
+    """
+
+    SUBCOMMAND = insights.SUBCOMMAND
+    ACTION = insights.ADD_LOGIN
+
+    def __init__(self, subparsers):
+        """Create command."""
+        # pylint: disable=no-member
+        CliCommand.__init__(self, self.SUBCOMMAND, self.ACTION,
+                            subparsers.add_parser(self.ACTION), None,
+                            None, [])
+        self.parser.add_argument(
+            '--username', dest='username', metavar='USERNAME',
+            help=_(messages.INSIGHTS_ADD_USERNAME_USER_HELP), required=True
+        )
+        self.parser.add_argument(
+            '--password', dest='password', metavar='PASSWORD',
+            help=_(messages.INSIGHTS_ADD_PASS_USER_HELP), required=True
+        )
+
+    def _do_command(self):
+        """Persist insights login configuration."""
+        login_config = {
+            'username': self.args.username,
+            'password': self.args.password,
+        }
+        write_insights_login_config(login_config)
+        print(messages.INSIGHTS_LOGIN_CONFIG_SUCCESS)
```

Comment:
I'd prefer logging instead of print. We could later control logger verbosity.
```
# at the begging of this file initialize logger
from logging import getLogger
log = getLogger('qpc')
# then here 
log.info(messages.INSIGHTS_LOGIN_CONFIG_SUCCESS)
```